Anatoly Shlom/flex/css/flex.less

```diff
@@ -0,0 +1,193 @@
+    font-weight: 300;
+    color: #ffffff;
+    padding: 0;
+    margin: 0;
+}
+
+/*Добавляем фон и немножко кросбраузерности)*/
+html { 
+    background: url("../img/bg.png") no-repeat center center fixed; 
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+  }
+
+/*оформляем сам виджет*/
+.widget {
+    background: no-repeat;
+    background-image: url("../img/main_bg.png");
+    background-size: cover; 
+    margin: 7% 11% 7% 11%;
+    padding: 10% 0 0 0;
+}
+
+/*верхняя часть виджета*/
+@media (max-width: 600px) {
+    .widget-data {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+    }
+}
+@media only screen and (min-width: 600px) {
+    .widget-data {
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-end;
+        align-items: flex-start;
+        margin:0 10% 0 0;
+    }
+}
+/*секция с текущей датой*/    
+@media only screen and (min-width: 600px) {
+    .current-date {
+        margin: 20px 20px 0 0 ;
+    }
+}
+.current-date__week-day {
+    font-size: 30px;
+}
+.current-date__day {
+    font-size: 14px;
+}
+
+/*секция с температурой,локацией,ветром*/
+.weather-block {
+    display: table-cell;
+    text-align: center;
+}
+.weather-block__temperature {
+    font-size: 92px;
+    font-weight: 700;
+}
+.weather-block__location {
+    font-size: 22px;
+}
+.wind-block {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: flex-end;
+}
+
+.wind-block__speed {
+    font-size: 29px;
+    font-weight: 400;
+}
+.wind-block__speed-format {
+    font-size: 14px;
+    font-weight: 400;
+    padding: 0 0 3px 0; 
+    /*три пикселя выше нужны что бы стояли на одной линии значение и единицы измерения
+    Верю в то что это баг шрифта, а не мои кривые руки)*/
+}
+
+
+/* Оформляем навбар*/
+
+.nav-block {
+    background:rgba(0,0,0,0.5);
+    margin: 10% 0 0 0;
+}
+
+@media (max-width: 600px) {
+    ul {
+        list-style: none; 
+        margin: 0 20px 0 20px; 
+        padding-left: 0; 
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        li {
+            font-size: 14px; 
+           
+        }
+    }
+
+    .nav-block__arrow-box {
+        display:flex;
+        justify-content: center;
+    }
+    .nav-block__divider_active {
+        padding-top: 0.3em; 
+        padding-bottom: 0.3em;
+        border-top: 1px solid; 
+        display: flex;
+        flex-direction: row;  
+        justify-content: space-around;
+     
+    }
+    .nav-block__divider_disable {  
+        display: flex;
+        flex-direction: row;
+        justify-content: space-around;
+
+    }
+    .nav-block__icon {
+        height: 20px;
+        margin-left: 20%;
+        margin-right: 20%;
+        align-self: center;
+    }
+    .nav-block__date {
+        width: 33%;
```

Comment:
You may set fix width for the text box, to prevent this 
<img width="420" alt="Screenshot 2019-07-20 at 13 14 13" src="https://user-images.githubusercontent.com/31970322/61577480-80b92100-aaf0-11e9-9aa7-cd7899481b16.png">

Reply:
Somehow changing to `width: 100px;` didn't fix the issue( what am i doing wrong? 
fix attempt here [https://github.com/epam-js-jun-2019/css-responsive-2/pull/8/commits/3dee598d5ec482b584706e93cff5c497247e07b9](https://github.com/epam-js-jun-2019/css-responsive-2/pull/8/commits/3dee598d5ec482b584706e93cff5c497247e07b9)

Reply:
Nice